[WIP] Peer review chapter (#2180)
alan-turing-institute/the-turing-way: book/website/communication/peer-review/peer-review-guidance.md

(cm-pr-guidance)=
# Guidance on Peer Review for a Journal

The text is inspired by this [blog post](https://renebekkers.wordpress.com/2020/06/24/how-to-review-a-paper/) by [Rene Bekkers](https://research.vu.nl/en/persons/rene-bekkers), released under the CC BY 4.0 licence.
There is no consensus among journals and editors on a set of minimum criteria that the peer review process should cover. 
The extensive set of criteria provided here are general considerations from the perspectives of opennnes and reproducibility.
The bar is set pretty high: a review checking each of the criteria will take you at least three hours, but more likely between five and eight hours. 

(cm-pr-guidance-before)=

> ## Before you accept the invitation

Reviewer: ```suggestion
## Before you accept an invitation to peer review
```

The following steps will help you make a decision whether to review the paper or not.
They need to be taken fast. 
It is a good practice to let the editor know immediately if you are unable to review a paper, so the editor can start to look for someone else to review the paper. 
Suggesting other possible reviewers is very helpful. 
Your non-response means a delay for the authors and the editor.

1. Check what kind of journal this is. 
Have you heard of it before?

> Does it have a reputaiton in your discipline?

Reviewer: ```suggestion
Does it have a reputation in your discipline?
```

You can set a requirement for the author to make the data available for peer review or openly available before you accept an invitation to review their work. 
It needs to be taken into account that some data  would be illegal or dangerous to make available, such as datasets that contain identifying information that cannot be removed. 
In most cases, the authors can provide at least partial access to the data by excluding variables that contain personal information.
If the paper does not provide a link to the data and the analysis script, you can ask the editor to ask the authors to provide the data and the code. 

You might get requests to review a paper that you have reviewed before, for a conference or another journal. 
In these cases, you can let the editor know and ask the editor whether she would like to see the previous review. 
For the editor it will be useful to know whether the current manuscript is the same as the version, or includes revisions.

(cm-pr-guidance-writing)=
## Writing a review

### How to organize the review work

> There is no established procedure, you need to find what works for you. 

Reviewer: ```suggestion
There is no established procedure, you need to find what works for you. The journal you are reviewing for may also provide guidance on how to write a review.
```

If you are reviewing for the first time, this workflow might be useful.
1. It might be a good idea to read a paper twice. 
First, go over the paper superficially and quickly. This gets you a sense of where the authors are going. 
2. After a quick glance at the paper, you can determine whether you think the paper has the potential to be published, perhaps after revisions. 
If you think the paper is beyond repair, explain this to the editor.
3. If the paper is good enough to be revised and resubmitted, you can start providing more detailed comments. 
3. The second time you go over the paper, do a very close reading. 
Some of the information may be in the supplementary information provided with the paper - read them carefully as well.

**The tone of the review**

> There is a ‘golden rule’ for people writign a review: treat others as you would like to be treated. 

Reviewer: ```suggestion
There is a ‘golden rule’ for people writing a review: treat others as you would like to be treated. 
```

It will allow the editor to adjudicate the review.
2. Then give a brief overview of the paper. 
If the invitation asks you to provide a general recommendation, state it early in the review.
The recommendation is the first thing the editor wants to know.
Typically, you are invited to recommend ‘reject’, ‘revise & resubmit’ – with major or minor revisions, or ‘accept’. 
3. Use ‘reject’ only if absolutely necessary.
Most journal editors make use of desk-rejection if the paper seems a poor fit; if the editor sent the paper out for review, she probably thinks it has the potential to be published.
Start from the assumption that the authors have invested a great deal of time in the paper and that they want to improve it. 
4. List the strengths and the weaknesses of the paper. 
To ease the message you can use the sandwich principle: start with the strengths, then discuss the weaknesses, and conclude with an encouragement.
5. Try to give actionable advice. 
For the weaknesses in the paper, suggest ways to repair them. 
It is convenient for both the editor and the authors to number your suggestions.

> Distinguish major issues such as not discussing alternative explanations from minor issues such as missing references and typos. 

Reviewer: ```suggestion
Distinguish major issues, such as not discussing alternative explanations, from minor issues such as missing references and typos. 
```

There could be points that the authors are underselling. 
In that case, you could identify them as strengths that the authors can emphasize more strongly.
It is handy to refer to issues with direct quotes and page numbers or line numbers. 
6. Some reviewers sign their reviews. 

> A non-anonymous review can be seen an accountability device: by exposing who they are to the authors of the paper, the reviewrs set higher standards for themselves.

Reviewer: ```suggestion
A non-annonymous review can be seen an accountability device: by exposing who they are to the authors of the paper, the reviewers set higher standards for themselves.
```

> There are [discussions](??? any reference here) about implementing open peer review model in research more generally.

Reviewer: I think you need to make clear here the difference between open peer review (which can be defined as the review being open for anyone viewing the paper to see) and signed peer review (i.e the reviewer signing their name). And that you can have a blend of either- i.e not being able to see the review but seeing the name of reviewer who did it, being able to see the review but not the name of the reviewer, or being able to see both!

Author: awesome, added this part!

(cm-pr-guidance)=
# Guidance on Peer Review for a Journal

The text is inspired by this [blog post](https://renebekkers.wordpress.com/2020/06/24/how-to-review-a-paper/) by [Rene Bekkers](https://research.vu.nl/en/persons/rene-bekkers), released under the CC BY 4.0 licence.
There is no consensus among journals and editors on a set of minimum criteria that the peer review process should cover. 

> The extensive set of criteria provided here are general considerations from the perspectives of opennnes and reproducibility.

Reviewer: ```suggestion
The extensive set of criteria provided here are general considerations from the perspectives of openness and reproducibility.
```
